docs/resources/netflow_exporter_policy.md

```diff
@@ -0,0 +1,189 @@
+subcategory: "Tenant Policies"
+layout: "aci"
+page_title: "ACI: aci_netflow_exporter_policy"
+sidebar_current: "docs-aci-resource-aci_netflow_exporter_policy"
+description: |-
+  Manages ACI Netflow Exporter Policy
+---
+
+# aci_netflow_exporter_policy #
+
+Manages ACI Netflow Exporter Policy
+
+
+
+## API Information ##
+
+* Class: [netflowExporterPol](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/netflowExporterPol/overview)
+
+* Supported in ACI versions: 2.2(1k) and later.
+
+* Distinguished Name Formats:
+  - `uni/infra/exporterpol-{name}`
+  - `uni/tn-{name}/exporterpol-{name}`
+
+## GUI Information ##
+
+* Locations:
+  - `Tenants -> Policies -> NetFlow -> NetFlow Exporters`
+  - `Fabric -> Access Policies -> Policies -> Interface -> NetFlow -> NetFlow Exporters`
+
+## Example Usage ##
+
+The configuration snippet below creates a Netflow Exporter Policy with only required attributes.
+
+```hcl
+
+resource "aci_netflow_exporter_policy" "example_tenant" {
+  parent_dn              = aci_tenant.example.id
+  destination_ip_address = "2.2.2.1"
+  destination_port       = "https"
+  name                   = "netfow_exporter"
+}
+
+```
+The configuration snippet below shows all possible attributes of the Netflow Exporter Policy.
+
+!> This example might not be valid configuration and is only used to show all possible attributes.
+
+```hcl
+
+resource "aci_netflow_exporter_policy" "full_example_tenant" {
+  parent_dn              = aci_tenant.example.id
+  annotation             = "annotation"
+  description            = "description_1"
+  qos_dscp_value         = "AF11"
+  destination_ip_address = "2.2.2.1"
+  destination_port       = "https"
+  name                   = "netfow_exporter"
+  name_alias             = "name_alias_1"
+  owner_key              = "owner_key_1"
+  owner_tag              = "owner_tag_1"
+  source_ip_type         = "custom-src-ip"
+  source_ip_address      = "1.1.1.1/10"
+  version                = "v9"
+  relation_to_vrf = [
+    {
+      annotation = "annotation_1"
+      target_dn  = "uni/tn-test_tenant/ctx-test_vrf"
+    }
+  ]
+  relation_to_epg = [
+    {
+      annotation = "annotation_1"
+      target_dn  = "uni/tn-test_tenant/ap-test_ap/epg-test_epg"
+    }
+  ]
+  annotations = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+  tags = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+}
+
+```
+
+All examples for the Netflow Exporter Policy resource can be found in the [examples](https://github.com/CiscoDevNet/terraform-provider-aci/tree/master/examples/resources/aci_netflow_exporter_policy) folder.
+
+## Schema ##
+
+### Required ###
+
+* `destination_ip_address` (dstAddr) - (string) The destination IP address of the remote node.
+* `destination_port` (dstPort) - (string) The destination port of the remote node.
+  - Valid Values: `dns`, `ftpData`, `http`, `https`, `pop3`, `rtsp`, `smtp`, `ssh`, `unspecified`.
```

Comment:
Here it seems to be missing the semantic equal port integer values?

Reply:
fixed.